source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/products/index.tsx

```diff
@@ -17,28 +24,56 @@ export interface Params {
 export interface IProps {
   match: Match<Params>;
   update: (payload: OrganizationAttributes) => void;
-  organization: ResourceObject<ORGANIZATIONS_TYPE, OrganizationAttributes>;
+  updateProductDefinition: (payload: ProductDefinitionResource) => void;
+  organization: OrganizationResource;
+  organizationProductDefinitions: OrganizationProductDefinitionResource[];
 }
 
-class ProductsRoute extends React.Component<IProps & i18nProps> {
+type IOwnProps =
+  & IProps
+  & i18nProps;
+
+const mapRecordsToProps = (passedProps) => {
+  const { organization } = passedProps;
+  const { type, id } = organization;
+
+  return {
+    organizationProductDefinitions: q => q.findRelatedRecords({ type, id }, 'organizationProductDefinitions')
+  };
+};
+
+class ProductsRoute extends React.Component<IOwnProps> {
+
   togglePrivacy = () => {
     const { update, organization } = this.props;
     const { makePrivateByDefault } = organization.attributes;
 
     update({ makePrivateByDefault: !makePrivateByDefault });
   }
 
+  updateProductDefinitionList = (productDefinition) => {
```

Comment:
this method name feels misleading. It says it's updating a list, but it only updates one product definition? :confused: